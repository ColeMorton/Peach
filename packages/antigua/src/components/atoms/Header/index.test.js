import React from 'react'
import renderer from 'react-test-renderer';
import Component from '.'

it('renders correctly', () => {
  const tree = renderer
    .create(<Component>Welcome to React components</Component>)
    .toJSON()
  expect(tree).toMatchSnapshot()
});
