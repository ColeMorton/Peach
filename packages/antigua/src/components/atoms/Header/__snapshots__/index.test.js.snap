// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui huge header"
>
  Welcome to React components
</div>
`;
